src/proposals/20240306/SparkEthereum_20240306.t.sol

```diff
@@ -0,0 +1,447 @@
+
+        _assertSupplyCapConfig({
+            asset:            RETH,
+            max:              80_000,
+            gap:              10_000,
+            increaseCooldown: 12 hours
+        });
+
+        _assertBorrowCapConfig({
+            asset:            RETH,
+            max:              2_400,
+            gap:              100,
+            increaseCooldown: 12 hours
+        });
+
+        /************/
+        /*** sDAI ***/
+        /************/
+
+        _assertSupplyCapConfig({
+            asset:            SDAI,
+            max:              1_000_000_000,
+            gap:              50_000_000,
+            increaseCooldown: 12 hours
+        });
+
+        _assertBorrowCapConfigNotSet(SDAI);
+
+        /************/
+        /*** USDC ***/
+        /************/
+
+        _assertSupplyCapConfigNotSet(USDC);
+
+        _assertBorrowCapConfig({
+            asset:            USDC,
+            max:              57_000_000,
+            gap:              6_000_000,
+            increaseCooldown: 12 hours
+        });
+
+        /************/
+        /*** USDT ***/
+        /************/
+
+        _assertSupplyCapConfigNotSet(USDT);
+
+        _assertBorrowCapConfig({
+            asset:            USDT,
+            max:              28_500_000,
+            gap:              3_000_000,
+            increaseCooldown: 12 hours
+        });
+
+        /************/
+        /*** WBTC ***/
+        /************/
+
+        _assertSupplyCapConfig({
+            asset:            WBTC,
+            max:              5_000,
+            gap:              500,
+            increaseCooldown: 12 hours
+        });
+
+        _assertBorrowCapConfig({
+            asset:            WBTC,
+            max:              2_000,
+            gap:              100,
+            increaseCooldown: 12 hours
+        });
+
+        /************/
+        /*** WETH ***/
+        /************/
+
+        _assertSupplyCapConfig({
+            asset:            WETH,
+            max:              2_000_000,
+            gap:              150_000,
+            increaseCooldown: 12 hours
+        });
+
+        _assertBorrowCapConfig({
+            asset:            WETH,
+            max:              1_000_000,
+            gap:              10_000,
+            increaseCooldown: 12 hours
+        });
+
+        /**************/
+        /*** wstETH ***/
+        /**************/
+
+        _assertSupplyCapConfig({
+            asset:            WSTETH,
+            max:              1_200_000,
+            gap:              50_000,
+            increaseCooldown: 12 hours
+        });
+
+        _assertBorrowCapConfig({
+            asset:            WSTETH,
+            max:              3_000,
+            gap:              100,
+            increaseCooldown: 12 hours
+        });
+    }
+
+    function testCapAutomatorCapUpdates() public {
+        GovHelpers.executePayload(vm, payload, executor);
+
```

Comment:
Can we change this to iterate through all assets in the reserves list? Should still work

Reply:
done